networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc

```diff
@@ -36,6 +36,14 @@ include::modules/virt-example-ethernet-nncp.adoc[leveloffset=+2]
 
 include::modules/virt-example-nmstate-multiple-interfaces.adoc[leveloffset=+2]
 
+include::modules/virt-example-host-vrf.adoc[leveloffset=+2]
```

Comment:
The example contains the VRF part but also more (setting routes specific to the egress machinery) so it feels odd seeing it in the nmstate section. We either add a simplified version of it, or we remove it

Reply:
I removed the routes section of the example CR. Let me know if any issues with this.